dashboard/app/mailers/pd/application/facilitator1819_application_mailer.rb

```diff
@@ -8,7 +8,29 @@ def confirmation(facilitator_application)
 
       mail(
         to: @application.user.email,
-        subject: 'We’ve received your facilitator application!'
+        subject: "We've received your facilitator application!"
+      )
+    end
+
+    def declined(facilitator_application)
+      raise "Unexpected #{facilitator_application.class}" unless facilitator_application.is_a? Facilitator1819Application
```

Comment:
note: I [removed](https://github.com/code-dot-org/code-dot-org/pull/19486) these checks on the other mailer methods, because they seemed unnecessary. They were only called by the relevant controller, or in this case cron job. I don't feel too strongly either way on this, but we should be consistent.